SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
```

Comment:
I don't think that this makes sense as we need independent security policies for different projects, like the backend provides different support than the front-end and the branch names and deployment addresses are different.

Reply:
Yes, showcasing a security.md in all repository is a good thing. I think we should also add security.md here as the more place it will be in, the better.
Also,any security.md do redirect to `security@puroto.net`

Reply:
hmm, I guess this is fine for now... We'll discuss that before we go live. People can't report vulnerabilities in the issues section so we have find a well way to do that anyway.